`xtask` add more features (#719)
extendr/extendr: xtask/README.md

Commands:
  check-fmt      Run cargo fmt on extendr
  fmt            Run `cargo fmt` on extendr crates
  r-cmd-check    Run R CMD check on {extendrtests}
  doc            Generate documentation for all features
  msrv           Check that the specified rust-version is MSRV
  devtools-test  Run devtools::test() on {extendrtests}

>   document       Generate wrappers by `rextendr::document`

Reviewer: Let's be consistent here, if you apply ``` `` ``` to code, change all other cases in the list. Also, when mentioning a function, don't forget a `()`

Reviewer: Something like this
```suggestion
  document       Generate wrappers with `rextendr::document()`
```

This checks if the rust code follows the specified `rustfmt`. It does not
format the code. In order to do so, please run `cargo fmt`.

## `fmt`

> This command calls `cargo fmt` in the workspace, and ensures that `cargo fmt` is called within `tests/extendrtests/src/rust` aswell.

Reviewer: ```suggestion
This command calls `cargo fmt` within the workspace, ensuring that the contents of `tests/extendrtests/src/rust` folder are formatted as well.
```
